forms-flow-api/src/formsflow_api/resources/application.py

```diff
@@ -431,6 +432,59 @@ def get(form_id: str):
         )
 
 
+@cors_preflight("GET,OPTIONS")
+@API.route("/formid/<string:form_id>/count", methods=["GET", "OPTIONS"])
+class ApplicationResourceCountByFormId(Resource):
+    """Resource for getting applications count on formid."""
+
+    @staticmethod
+    @auth.require
```

Comment:
`@auth.require` can be removed since already using `@auth.has_one_of_roles([DESIGNER_GROUP])`